base/nullable.jl

```diff
@@ -63,16 +63,79 @@ get(x::Nullable) = x.isnull ? throw(NullException()) : x.value
 
 isnull(x::Nullable) = x.isnull
 
-function isequal(x::Nullable, y::Nullable)
-    if x.isnull && y.isnull
-        return true
-    elseif x.isnull || y.isnull
-        return false
+
+## Operators
+
+"""
+    null_safe_op(f::Any, ::Type)::Bool
```

Comment:
I guess this is a combination of three things:

- the operation is expected to be not much more expensive than a branch, so the optimization is reasonably profitable
- the type is pointer free and so cannot be #undef
- the operation is supported and will not fail on all patterns of bits possible

so perhaps `isbitstotal`? Like `isbits`, but also total under a certain operation?